src/routes/v2/authenticated/review.ts

```diff
@@ -125,12 +129,23 @@ export class ReviewsRouter {
 
     return this.sitesService
       .getStagingUrl(userWithSiteSessionData)
-      .andThen((stagingLink) =>
-        this.reviewRequestService.compareDiff(
+      .andThen((stagingLink) => {
+        if (
+          userWithSiteSessionData.growthbook?.getFeatureValue(
+            FEATURE_FLAGS.IS_LOCAL_DIFF_ENABLED,
+            false
+          )
+        ) {
+          return this.reviewRequestService.compareDiffLocal(
+            userWithSiteSessionData,
+            stagingLink
+          )
+        }
+        return this.reviewRequestService.compareDiff(
           userWithSiteSessionData,
           stagingLink
         )
```

Comment:
hmm this logic seems like it should live in the `service` level rather than at the router level. 

could we refactor this such that this lives in `reviewRequestService` instead?

Reply:
[00fa655](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/00fa6554e11a223043fcadb139e25e1154cdaa1f)